scripts/travis.sh

```diff
@@ -10,6 +10,7 @@
   cargo test --features test --package gluon_vm &&
   cargo test --features test --package gluon_format &&
   cargo test --features test --package gluon &&
+  cargo test --features test --package gluon_test &&
```

Comment:
I want to highlight that I added a separate crate for the gluon script tests here https://github.com/gluon-lang/gluon/pull/402/commits/3b7073f2fb5e4f9130f49b702200de08743c2b6c . Still undecided which way is the right approach here...